pkgs/top-level/aliases.nix

```diff
@@ -527,7 +527,7 @@ mapAliases {
   bitcoind-unlimited = throw "bitcoind-unlimited has been removed as it was broken and unmaintained"; # Added 2024-07-15
   bird = throw "The bird alias was ambiguous and has been removed for the time being. Please explicitly choose bird2 or bird3."; # Added 2025-01-11
   bisq-desktop = throw "bisq-desktop has been removed because OpenJFX 11 was removed"; # Added 2024-11-17
-  bitmeter = throw "bitmeter has been removed because it relies on gtk2 and upstream is abandoned since 2006"; # Aded 2025-10-03
+  bitmeter = throw "bitmeter has been removed because it relies on gtk2 and upstream is abandoned since 2006, use x42-meter instead."; # Aded 2025-10-03
```

Comment:
I concerned that `x42-meter` is not a package itself but from `x42-plugins`, so it could create a bit of a friction (i tried to `nix-shell -p` it myself too 😅)

Reply:
Ah, yes, sorry.
So the workflow is:
- install x42-plugins
- run ``x42-meter 18``  (there are many meters in this package, 18 is the bitmeter)

How should I word the alias?

Reply:
that's the difficult part :eyes:

Reply:
OK, done.
Also: Aded -> Added